src/main/steps/applicant1/hub-page/progress-bar.njk

```diff
@@ -1,45 +1,33 @@
 {% from "components/hmcts-progress-bar/macro.njk" import hmctsProgressBar %}
 
 {% macro textWithVisuallyHiddenSpan(text, isComplete) %}
-  {{text}}<span class="govuk-visually-hidden">{{'' if isComplete else 'not '}}completed</span>
+  {{text}} <span class="govuk-visually-hidden">{{'' if isComplete else 'not '}}completed</span>
 {% endmacro %}
 
 {% if isJointApplication %}
   {% set items = [{
     label: {
-      html: textWithVisuallyHiddenSpan(applicationSubmitted, true)
+      html: textWithVisuallyHiddenSpan(submitted, true)
     },
     complete: true,
     active: false
   }, {
     label: {
-      html: textWithVisuallyHiddenSpan(conditionalOrderApplication, displayState.isAtOrAfter('AwaitingLegalAdvisorReferral'))
-    },
-    complete: displayState.isAtOrAfter('AwaitingLegalAdvisorReferral'),
-    active: hasApplicantConfirmedReceipt
-  }, {
-    label: {
-      html: textWithVisuallyHiddenSpan(conditionalOrderGranted, displayState.isAtOrAfter('ConditionalOrderPronounced'))
+      html: textWithVisuallyHiddenSpan(conditionalOrder, displayState.isAtOrAfter('AwaitingLegalAdvisorReferral'))
     },
     complete: displayState.isAtOrAfter('ConditionalOrderPronounced'),
-    active: displayState.state() == 'AwaitingPronouncement'
-  }, {
-    label: {
-      html: textWithVisuallyHiddenSpan(finalOrderApplication, displayState.isAtOrAfter('FinalOrderRequested'))
-    },
-    complete: displayState.isAtOrAfter('FinalOrderRequested'),
-    active: displayState.state() == 'AwaitingFinalOrder'
+    active: hasApplicantConfirmedReceipt
```

Comment:
won't this make it active any time after they confirm their receipt? Which doesn't seem right, as it shouldn't be active when finalOrder is

Reply:
I've noticed that this is how it was before, but still confused by it

Reply:
good point, thanks, ill have a look